roles/confluent.variables/defaults/main.yml

```diff
@@ -106,8 +106,14 @@ archive_config_base_path: "{{ archive_destination_path }}"
 ### Boolean to have cp-ansible download the Confluent CLI
 confluent_cli_download_enabled: "{{rbac_enabled or secrets_protection_enabled}}"
 
-### Full path on hosts to install the Confluent CLI
-confluent_cli_path: /usr/local/bin/confluent
+### The path the Confluent CLI archive is expanded into.
+confluent_cli_base_path: /opt/confluent-cli
+
+### Full path on hosts to Confluent CLI executable
+confluent_cli_path: "{{confluent_cli_base_path}}/{{confluent_cli_binary}}/{{confluent_cli_binary}}"
```

Comment:
The path to the binary went from `/usr/local/bin/confluent` to `/opt/confluent-cli/confluent/confluent` ?

I think I prefer it being in /usr/local/bin because when I sshd to hosts it was in my PATH

Reply:
That's because the archive structure contains a folder named `confluent` when you uncompress it, so using a simple `unarchive` operation in `/usr/local` will result to have the executable in `/usr/local/confluent/confluent`. This can possibly be solved using `confluent_cli_base_path: /opt/confluent-cli` and `confluent_cli_path: /usr/local/bin/confluent` and add an additional task that copy or symlink the executable in that position for backward compatibility and having it in the PATH when using ssh.

Reply:
I see... Well if unarchive does not work it can always be reverted to the `shell` module. It is nice how the unarchive allows for the "tar on control node" feature though, but I suppose that could be handled with a simple `copy` task too.

I am thinking about a copy or symlink myself. The install.sh script does not "leave anything behind" right? It just downloads the binary where you say you want it. 

One question I have is: Do we want this directory to be created in opt or do we simply want the binary created at the `confluent_cli_path`? 

One case for the directory (I guess) is if you want to start trying out multiple versions of the binary, but I'm not sure that's actually a popular use case.

Reply:
`install.sh` leaves behind some stuff, for example documentation and licensing information is left into a temporary folder (`log_info "NOTICE: see licenses located in ${tmpdir}/${BINARY}"`) and just copy the binary to the desired location.
As far as I know `/opt` should be the right place for optional packages ([FHS specs](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s13.html)) and I would say that we want to create a directory into /opt hierarchy for it. Symlinking the binary in the default path is not standard but is a common practice, and using a folder possibly allows you to keep multiple versions of the executable.

Reply:
I've added a symlink creation. This effectively unify the confluent cli installation from archive (remote or not) and custom download url if defined, placing both on the same /opt hierarchy and symlinking the executable to `confluent_cli_path` for convenience.